backend/src/data/sessionsDataAccess.ts

```diff
@@ -0,0 +1,53 @@
+import { pool } from '../config/db'
+import { v4 as uuidv4 } from 'uuid';
+import { ISession } from '../types/session';
+import { ISessionAccessCode } from '../types/sessionAccessCode';
+
+export const createSession = async (): Promise<ISession | null> => {
+    try {
+   
+  
+      const query = `
```

Comment:
We can choose 2 or 4-spaces for indentation, it should be consistent throughout the code 💅

Reply:
Good idea. I'll add add a linter to the todo list.